cves/kernel/CVE-2020-27673.yml

```diff
@@ -109,66 +109,66 @@ vcc_instructions: |
-- commit: 30978346372e5c43a652cfbd4533c6bd5427c33b
-  :note: Discovered automatically by archeogit.
-- commit: 589d03e93f6cd595f68891e48f0804f2c8f38aae
-  :note: Discovered automatically by archeogit.
-- commit: ab9a1cca3d172876ae9d5edb63abce7986045597
-  :note: Discovered automatically by archeogit.
-- commit: d0b075ffeede257342c3afdbeadd2fda8504ecee
-  :note: Discovered automatically by archeogit.
-- commit: 9a489f45a155fe96b9b55fbbef2b757ef7737cfc
-  :note: Discovered automatically by archeogit.
-- commit: 54c9de89895e0a36047fcc4ae754ea5b8655fb9d
-  :note: Discovered automatically by archeogit.
-- commit: 073d0552ead5bfc7a3a9c01de590e924f11b5dd2
-  :note: Discovered automatically by archeogit.
-- commit: 7beb290caa2adb0a399e735a1e175db9aae0523a
-  :note: Discovered automatically by archeogit.
-- commit: e46cdb66c8fc1c8d61cfae0f219ff47ac4b9d531
-  :note: Discovered automatically by archeogit.
-- commit: 6cb6537d34a146c8f3e232da9ffd79ae1265fed3
-  :note: Discovered automatically by archeogit.
-- commit: 38e20b07efd541a959de367dc90a17f92ce2e8a6
-  :note: Discovered automatically by archeogit.
-- commit: 1fe565517b57676884349dccfd6ce853ec338636
-  :note: Discovered automatically by archeogit.
-- commit: f87e4cac4f4e940b328d3deb5b53e642e3881f43
-  :note: Discovered automatically by archeogit.
-- commit: 7e186bdd0098b34c69fb8067c67340ae610ea499
-  :note: Discovered automatically by archeogit.
-- commit: '0891fb39ba67bd7ae023ea0d367297ffff010781'
-  :note: Discovered automatically by archeogit.
-- commit: 348be43384e6bcd5e9da7ff5f1680d49f65c488d
-  :note: Discovered automatically by archeogit.
-- commit: 229664bee6126e01f8662976a5fe2e79813b77c8
-  :note: Discovered automatically by archeogit.
-- commit: f731e3ef02b4744f4d7ca2f63539b900e47db31f
-  :note: Discovered automatically by archeogit.
-- commit: 0ec53ecf38bcbf95b4b057328a8fbba4d22ef28b
-  :note: Discovered automatically by archeogit.
-- commit: b5328cd14557880e9eb757a8a9c8a88f1b23533a
-  :note: Discovered automatically by archeogit.
-- commit: c149e4cd08ba01f4d2d0104f469d5f5419294e06
-  :note: Discovered automatically by archeogit.
-- commit: ca62ce8cde36cde5a31af8f9bb23e6430700e437
-  :note: Discovered automatically by archeogit.
-- commit: 94032c506720e26402db64987e08168565b57990
-  :note: Discovered automatically by archeogit.
-- commit: 9158c3588a0693ef11208f5961f0d6b0241694d8
-  :note: Discovered automatically by archeogit.
-- commit: 780f36d8b3fa9572f731d4fb85067b2e45e6f993
-  :note: Discovered automatically by archeogit.
-- commit: 3de88d622fd68bd4dbee0f80168218b23f798fd0
-  :note: Discovered automatically by archeogit.
-- commit: 0102e4efda76d0721bc744dd80258eb6cfa25fc3
-  :note: Discovered automatically by archeogit.
-- commit: c8761e2016aa51a2829563b02a0a55913bdb0be8
-  :note: Discovered automatically by archeogit.
-- commit: 4892c9b4ada9f9a71a0da7a268f95e988d88064b
-  :note: Discovered automatically by archeogit.
+  - commit: 2ec16bc0fc7ab544f2d405fd4fdd0d717c5ec0c5
```

Comment:
Holy cow! I didn't realize I had assigned one that was this much work. Thank you for confirming all these. I tried to filter out some of the big ones but I missed this one.

Reply:
Haha, sure! I hope they're accurate- some of them I was hesitant on and made a call.